src/io/flutter/sdk/FlutterSdkManager.java

```diff
@@ -50,7 +79,7 @@ public void removeListener(@NotNull Listener listener) {
   }
 
   private static boolean isGlobalFlutterSdkSet() {
-    return FlutterSdk.getGlobalFlutterSdk() != null;
+    return FlutterSdk.getGlobalFlutterSdk() != null && FlutterSdkUtil.isFluttering();
```

Comment:
Probably doesn't matter but I'd be inclined to rename the method to make it clear another test is happening.